crates/viewer/re_renderer/src/video/player.rs

```diff
@@ -121,13 +121,15 @@ impl VideoPlayer {
     pub fn frame_at(
         &mut self,
         render_ctx: &RenderContext,
-        presentation_timestamp_s: f64,
+        time_since_video_start_in_seconds: f64,
         video_data: &[u8],
     ) -> Result<VideoFrameTexture, VideoPlayerError> {
-        if presentation_timestamp_s < 0.0 {
+        if time_since_video_start_in_seconds < 0.0 {
             return Err(VideoPlayerError::NegativeTimestamp);
         }
-        let presentation_timestamp = Time::from_secs(presentation_timestamp_s, self.data.timescale);
+        let presentation_timestamp =
+            Time::from_secs(time_since_video_start_in_seconds, self.data.timescale)
+                + self.data.sample_statistics.minimum_presentation_timestamp;
```

Comment:
This feels quite error prone. It would be less error prone if the `minimum_presentation_timestamp` was an argument to `Time::from_secs/to_secs`

Reply:
good idea

Reply:
then again. Argh. It's so poorly defined.

Reply:
there.
https://github.com/rerun-io/rerun/pull/8029/commits/8ae876be380536fb23550a948d71f78e2d9d9b4c
not thrilled :/

Reply:
ah well now webcodecs is failing which _does_ highlight that I have to think that one through as well.....